src/keyValueValidator/functionsToValidate.js

```diff
@@ -1,73 +1,76 @@
-import { keyTypes } from '../utils/constants.js';
+import { cmcdTypes, errorTypes, keyTypes } from '../utils/constants.js';
 import { createError } from '../utils/error.js';
 
 export const checkMaxLength = (errors, key, value) => {
   if (value.length > 64) {
     console.log(`Invalid value for key '${key}'. Maximum length is 64 characters.`);
-    errors.push(createError('invalid-value', key, value));
+    errors.push(createError(errorTypes.invalidValue, key, value));
   }
 };
 
 export const isEncoded = (errors, key, value) => {
   if (decodeURIComponent(value) === value) {
     console.log(`The '${key}' value must be URLencoded.`);
-    errors.push(createError('parameter-encoding', key, value));
+    errors.push(createError(errorTypes.parameterEncoding, key, value));
   }
 };
 
 export const checkValidNrrFormat = (errors, key, value) => {
-  if (!(/^(\d*-\d*)$/.test(value)) || !(/^\d*-$/.test(value)) || !(/^-\d*$/.test(value))) {
-    console.log(`The '${key}' value format is incorrect`);
-    errors.push(createError('incorrect-format', key, value));
+  if (!(/^(\d*-\d*)$/.test(value)) && !(/^\d*-$/.test(value)) && !(/^-\d*$/.test(value))) {
+    console.log('/^(d*-d*)$/.test(value)', !(/^(\d*-\d*)$/.test(value)));
+    console.log('(/^d*-$/.test(value))', !(/^\d*-$/.test(value)));
+    console.log('(/^-d*$/.test(value))', !(/^-\d*$/.test(value)));
```

Comment:
Are these needed?

Reply:
No, we don´t need it. It just to debug. I deleted this lines